### PR TITLE
ImagesTable: Add alert for CentOS Stream 8 blueprints

### PR DESCRIPTION
**Pull request by regexowl**
This adds an alert about CentOS Stream 8 being EOL.

### CODE REVIEW

File: src/Components/ImagesTable/ImagesTableToolbar.tsx [hunk 1]
```diff
@@ -141,6 +150,20 @@ const ImagesTableToolbar: React.FC<imagesTableToolbarProps> = ({
...
+          blueprintsComposes.data.length > 0 &&
+          isBlueprintDistroCentos8() && (
+            <Alert
+              style={{
+                margin:
+                  '0 var(--pf-v5-c-toolbar__content--PaddingRight) 0 var(--pf-v5-c-toolbar__content--PaddingLeft)',
+              }}
+              isInline
+              variant="warning"
+              title="CentOS Stream 8 is EOL, building images from this blueprint will result in failure."
```
Comment (ochosi) [inline src/Components/ImagesTable/ImagesTableToolbar.tsx:163]:
```suggestion
              title="CentOS Stream 8 is end of life, building images from this blueprint will fail."
```

File: src/test/Components/Blueprints/Blueprints.test.tsx [hunk 2]
```diff
@@ -120,6 +121,22 @@ describe('Blueprints', () => {
     ).not.toBeInTheDocument();
   });
 
+  test('CentOS 8 Stream renders', async () => {
+    renderWithReduxRouter('', {});
+
+    await selectBlueprintById(blueprintIdCentos8);
+    await screen.findByText(
+      'CentOS Stream 8 is EOL, building images from this blueprint will result in failure.'
```
Comment (ochosi) [inline src/test/Components/Blueprints/Blueprints.test.tsx:129]:
```suggestion
      'CentOS Stream 8 is end of life, building images from this blueprint will fail.'
```

File: src/test/Components/Blueprints/Blueprints.test.tsx [hunk 3]
```diff
@@ -120,6 +121,22 @@ describe('Blueprints', () => {
...
+    );
+
+    await selectBlueprintById(blueprintIdWithComposes);
+    expect(
+      screen.queryByText(
+        'CentOS Stream 8 is EOL, building images from this blueprint will result in failure.'
```
Comment (ochosi) [inline src/test/Components/Blueprints/Blueprints.test.tsx:135]:
```suggestion
        'CentOS Stream 8 is end of life, building images from this blueprint will fail.'
```

**Review by ochosi — APPROVED** [review]
LGTM but I'd not use acronyms - not everyone may be familiar with them.

**Review by ochosi — APPROVED** [review]

File: src/Components/ImagesTable/ImagesTableToolbar.tsx (hunk 1 above)
Comment (regexowl) [reply src/Components/ImagesTable/ImagesTableToolbar.tsx]: Thank you! I've updated the wording.

File: src/Components/ImagesTable/ImagesTableToolbar.tsx [hunk 4]
```diff
@@ -141,6 +150,20 @@ const ImagesTableToolbar: React.FC<imagesTableToolbarProps> = ({
...
+          blueprintsComposes.data.length > 0 &&
+          isBlueprintDistroCentos8() && (
+            <Alert
+              style={{
+                margin:
+                  '0 var(--pf-v5-c-toolbar__content--PaddingRight) 0 var(--pf-v5-c-toolbar__content--PaddingLeft)',
+              }}
+              isInline
+              variant="warning"
+              title="CentOS Stream 8 is end of life, building images from this blueprint will fail."
```
Comment (ondrejbudai) [inline src/Components/ImagesTable/ImagesTableToolbar.tsx:163]: What about? ```suggestion
              title="CentOS Stream 8 is no longer supported, building images from this blueprint will fail."
```

I think I live the wording more, wdyt? We can also link to the announcement, but if that's too hard, it's not needed: https://blog.centos.org/2023/04/end-dates-are-coming-for-centos-stream-8-and-centos-linux-7/

Comment (regexowl) [reply src/Components/ImagesTable/ImagesTableToolbar.tsx]: Updated the wording :+1: 

Not sure about the link. I think the only links we currently use lead to documentation or other services. :thinking: I'm not against adding it, but I'm not sure if it's okay to link "outside".

**Pull request merged**